<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-EUDIV0F" name="com.petStore.test.nonfuctional.GetUserByNameNonFucntional" tests="2" failures="1" timestamp="11 Sep 2022 20:38:23 GMT" time="0.512" errors="0">
  <testcase name="test_getUserForNonExistentUserNameWithSqlInjection" time="0.246" classname="com.petStore.test.nonfuctional.GetUserByNameNonFucntional">
    <failure type="java.lang.AssertionError" message="Processing should fail with Error Code 400 as Payload rather than processing giving 404 not found search result as we had sql injection in query paramaters : expected [400] but found [404]">
      <![CDATA[java.lang.AssertionError: Processing should fail with Error Code 400 as Payload rather than processing giving 404 not found search result as we had sql injection in query paramaters : expected [400] but found [404]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:372)
	at com.petStore.test.nonfuctional.GetUserByNameNonFucntional.test_getUserForNonExistentUserNameWithSqlInjection(GetUserByNameNonFucntional.java:95)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
]]>
    </failure>
  </testcase> <!-- test_getUserForNonExistentUserNameWithSqlInjection -->
  <testcase name="test_getUserForNonExistentUserName" time="0.266" classname="com.petStore.test.nonfuctional.GetUserByNameNonFucntional"/>
</testsuite> <!-- com.petStore.test.nonfuctional.GetUserByNameNonFucntional -->
